installscripts/jazz-terraform-unix-noinstances/scripts/modifyCodebase.sh

```diff
@@ -14,29 +14,24 @@ sed -i "s/{inst_stack_prefix}/$stackprefix/g" ./jazz-core/serverless-config-pack
 
 #-------------------------------------------
 
-platform_services=("cognito-authorizer" "platform_logs" "platform_usermanagement" "platform-services-handler" "platform_events" "platform_services" "platform_logout" "platform_login" "cloud-logs-streamer" "is-service-available" "delete-serverless-service" "create-serverless-service" "platform_email" "platform_events-handler" )
+platform_services=("cognito-authorizer" "logs" "usermanagement" "services-handler" "events" "services" "logout" "login" "cloud-logs-streamer" "is-service-available" "delete-serverless-service" "create-serverless-service" "email" "events-handler" )
 servicename="_services_prod"
 tablename=$stackprefix$servicename
 timestamp=`date --utc +%FT%T`
 
 for element in "${platform_services[@]}"
 do
   uuid=`uuidgen -t`
-  echo -n > ./jazz-core/$element/deployment-env.yml
-  echo "service_id: "$uuid >> ./jazz-core/$element/deployment-env.yml
- 
-  if [[ $element =~ ^platform ]] ; then
-    service_name="${element:9}"
-  else
-    service_name=$element
-  fi
+  echo -n > ./jazz-core/jazz_$element/deployment-env.yml
```

Comment:
Once you maker the change suggested above, you can remove jazz_ here

Reply:
In cooperated the review comment